modules/trusted-firmware-m/tfm_boards/common/attest_hal.c

```diff
@@ -132,7 +148,24 @@ enum tfm_plat_err_t tfm_plat_get_boot_seed(uint32_t size, uint8_t *buf)
 	if (nrf_err != NRF_CC3XX_PLATFORM_SUCCESS) {
 		return TFM_PLAT_ERR_SYSTEM_ERR;
 	}
+#elif defined(CONFIG_CRACEN_HW_PRESENT)
+	if (boot_seed_not_set) {
+		psa_status_t  psa_err = psa_generate_random(boot_seed, sizeof(boot_seed));
+
+		if (psa_err != PSA_SUCCESS) {
+			return TFM_PLAT_ERR_SYSTEM_ERR;
+		}
+
+		boot_seed_not_set = false;
```

Comment:
Boot seed not set is false so not not set so that means... Hmm... Set? (please avoid double negation)